ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,156 +1,166 @@
-{%             if vlan_interfaces[vlan_interface].vrrp.priority is defined and vlan_interfaces[vlan_interface].vrrp.priority is not none and vlan_interfaces[vlan_interface].vrrp.priority > 1 %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].vrrp.virtual_router is arista.avd.defined %}
+{%         if vlan_interfaces[vlan_interface].vrrp.priority is arista.avd.defined %}
    vrrp {{ vlan_interfaces[vlan_interface].vrrp.virtual_router }} priority-level {{ vlan_interfaces[vlan_interface].vrrp.priority }}
-{%             endif %}
-{%             if vlan_interfaces[vlan_interface].vrrp.advertisement_interval is defined and vlan_interfaces[vlan_interface].vrrp.advertisement_interval is not none %}
+{%         endif %}
+{%         if vlan_interfaces[vlan_interface].vrrp.advertisement_interval is arista.avd.defined %}
    vrrp {{ vlan_interfaces[vlan_interface].vrrp.virtual_router }} advertisement interval {{ vlan_interfaces[vlan_interface].vrrp.advertisement_interval }}
-{%             endif %}
-{%             if vlan_interfaces[vlan_interface].vrrp.preempt_delay_minimum is defined and vlan_interfaces[vlan_interface].vrrp.preempt_delay_minimum is not none %}
+{%         endif %}
+{%         if vlan_interfaces[vlan_interface].vrrp.preempt_delay_minimum is arista.avd.defined %}
    vrrp {{ vlan_interfaces[vlan_interface].vrrp.virtual_router }} preempt delay minimum {{ vlan_interfaces[vlan_interface].vrrp.preempt_delay_minimum }}
-{%             endif %}
-{%             if vlan_interfaces[vlan_interface].vrrp.ipv4 is defined and vlan_interfaces[vlan_interface].vrrp.ipv4 is not none %}
+{%         endif %}
+{%         if vlan_interfaces[vlan_interface].vrrp.ipv4 is arista.avd.defined %}
    vrrp {{ vlan_interfaces[vlan_interface].vrrp.virtual_router }} ipv4 {{ vlan_interfaces[vlan_interface].vrrp.ipv4 }}
-{%             endif %}
-{%             if vlan_interfaces[vlan_interface].vrrp.ipv6 is defined and vlan_interfaces[vlan_interface].vrrp.ipv6 is not none %}
+{%         endif %}
+{%         if vlan_interfaces[vlan_interface].vrrp.ipv6 is arista.avd.defined %}
    vrrp {{ vlan_interfaces[vlan_interface].vrrp.virtual_router }} ipv6 {{ vlan_interfaces[vlan_interface].vrrp.ipv6 }}
-{%             endif %}
-{%           endif %}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_attached_host_route_export is defined and vlan_interfaces[vlan_interface].ip_attached_host_route_export is not none %}
+{%     endif %}
+{%     if vlan_interfaces[vlan_interface].ip_attached_host_route_export.distance is arista.avd.defined %}
    ip attached-host route export {{ vlan_interfaces[vlan_interface].ip_attached_host_route_export.distance | arista.avd.default("") }}
-{%         endif %}
-{%     endfor %}
-{% endif %}
+{%     elif vlan_interfaces[vlan_interface].ip_attached_host_route_export is arista.avd.defined %}
```

Comment:
Do we need this elif, if we set the default in the line 161 as empty using the new filter?

Reply:
using default to "" would render a blank space instead (which was what we had before), so this change is correct.